UPGRADE_GUIDE.md

```diff
@@ -1,5 +1,12 @@
 # Upgrade Guide
 
+## From `0.3.2` to `0.3.3`
+
+### Allow unsafe code in Generated Code
```

Comment:
Link to https://docs.unity3d.com/Manual/class-AssemblyDefinitionImporter.html which describes the various properties